### DISCUSSION TITLE
Reporting Full DashBoard is Not Coming

### DISCUSSION THREAD

**Discussion by code-marcin** [apache/superset#15682] [Q&A / Help]
Hello Team,

We are currently evaluating the Reports and Alerting Features of Superset, we configure them Alerts and Report.

But while sending reports the Screenshot send on mail is not a full dashboard, whole screenshot is not captured 

We are using chrome browser this is my supersetconfig file:
```
WEBDRIVER_TYPE = "chrome"
WEBDRIVER_OPTION_ARGS = [
    "--force-device-scale-factor=2.0",
    "--high-dpi-support=2.0",
    "--headless",
    "--disable-gpu",
    "--disable-dev-shm-usage",
    "--no-sandbox",
    "--disable-setuid-sandbox",
    "--disable-extensions",
]
```
Any help regarding this how to capture the full screenshot for the Reports ?

**Comment by nytai** [comment]
I think there's a setting related to the window size: `WEBDRIVER_WINDOW`

https://github.com/apache/superset/blob/070f0b6cb2a1c3385bc2a7ed2448aa7eb1251666/superset/config.py#L1046

**Reply by code-marcin** [reply]
Thanks @nytai  will try it once, but for large dashboards like 2-3 scrolls (20+ charts) what should be the ideal configuration for your usage what have u configured ?
